wsmaster/che-core-api-account/src/test/java/org/eclipse/che/account/spi/tck/AccountDaoTest.java

```diff
@@ -96,6 +96,14 @@ public void shouldThrowConflictExceptionWhenUpdatingAccountWithExistingName() th
     accountDao.update(account);
   }
 
+  @Test(expectedExceptions = ConflictException.class)
+  public void shouldThrowConflictExceptionWhenCreatingAccountWithExistingName() throws Exception {
+    AccountImpl account = accounts[0];
+    account.setName(accounts[1].getName());
+
+    accountDao.create(account);
```

Comment:
You try to create user with the existing name and id. To ensure that conflict is thrown because of names conflct it is needed to rewrite this test as
```
AccountImpl account = new AccountImpl(NameGenerator.generate("account", 5), accounts[0].getName(), "test");

accountDao.create(account);
```